apps/communications/contacts/views/details/details.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+    <meta charset="utf-8">
+    <meta name="theme-group" content="theme-communications">
+    <meta name="theme-color" content="var(--header-background)">
+    <title> Contacts </title>
+
+    <!-- Hack for avoiding 'glitch' when loading the view -->
+    <style type="text/css">
+        html, body {
+            background: #00b7b7 !important;
```

Comment:
No, we can't. But this will be removed in a near future once we have prerender =)